src/simulation/deviceInterface/tempMeasurement/tempMeasurement.rst

```diff
@@ -0,0 +1,111 @@
+Executive Summary
+-----------------
+Models the addition of noise, bias, and faults to temperature measurements.
+
+Message Connection Descriptions
+-------------------------------
+The following table lists all module input and output messages. The module msg connection 
+is set by the user from python. The msg type contains a link to the message structure definition, 
+while the description provides information on what this message is used for.
+
+.. list-table:: Module I/O Messages
+    :widths: 25 25 50
+    :header-rows: 1
+
+    * - Msg Variable Name
+      - Msg Type
+      - Description
+    * - tempInMsg
+      - :ref:`TemperatureMsgPayload`
+      - True temperature measurement
+    * - tempOutMsg
+      - :ref:`TemperatureMsgPayload`
+      - Sensed temperature measurement with corruptions
+
+Detailed Module Description
+---------------------------
+
+This module simulates the corruption of a true thermal measurement by noise,
+bias, and three faults:
+
+- "TEMP_FAULT_STUCK_VALUE" is faulty behavior where the measurement sticks to a specific value
```

Comment:
Change "TEMP_FAULT_STUCK_VALUE" to ``TEMP_FAULT_STUCK_VALUE`` to make these name be formatted as variables.  Do this for all options

Reply:
Done.